downstream/modules/automation-hub/proc-deploying-your-system-for-container-signing.adoc

```diff
@@ -19,9 +19,9 @@ As an Ansible Automation Platform user, you can now confirm if an EE/Container i
         automationhub_container_signing_service_script: _path/to/executable_
 
 
-. Navigate to automation hub
+. Navigate to automation hub.
 
-. Click *Signature Keys* in the left menu pane
+. Click btn:[Signature Keys] in the left menu pane.
```

Comment:
Using directional information is a violation and I think this is a menu selection, suggest:

In the navigation pane, select menu:Signature Keys[].